goaviatrix/controller.go

```diff
@@ -422,3 +430,54 @@ func (c *Client) SetControllerVpcDnsServer(enabled bool) error {
 		Action: action,
 	}, BasicCheck)
 }
+
+func ValidateBackupConfig(d *schema.ResourceData) error {
+	backupCloudType := d.Get("backup_cloud_type").(int)
+	backupAccountName := d.Get("backup_account_name").(string)
+	backupBucketName := d.Get("backup_bucket_name").(string)
+	backupStorageName := d.Get("backup_storage_name").(string)
+	backupContainerName := d.Get("backup_container_name").(string)
+	backupRegion := d.Get("backup_region").(string)
+
+	if backupCloudType == 0 || backupAccountName == "" {
+		err := fmt.Errorf("please specify 'backup_cloud_type' and 'backup_account_name'" +
+			" to enable backup configuration")
+		return err
```

Comment:
Why add the extra line to assign to a variable before returning? Does not seem more clear to me